design/remove-deprecated-topic-operator-from-kafka-crd.md

```diff
@@ -1,8 +1,8 @@
 # Remove deprecated Topic Operator deployment in Kafka CRD
 
-| State    |
-|----------|
-| Approved |
+| State       |
+|-------------|
+| Implemented |
```

Comment:
did we decide to have the state in the design doc at the end?

Reply:
The is not related to #3116 which is still open. This was there before the #3116 and the comments there.